### ISSUE TITLE
Test failure

### ISSUE THREAD

**Issue by erikgrinaker** [sanity-io/litter#15]
Running `go test` on master yields:

```
--- FAIL: TestSdump_customDumper (0.00s)
    --- FAIL: TestSdump_customDumper/customDumper (0.00s)
        Error Trace:    dump_test.go:213
    	            	dump_test.go:171
    	Error:      	Strings are different (left is expected, right is actual):
    	            	map[string]interface {}{					map[string]interface {}{
    	            	  "v1": *litter_test.CustomMultiLineDumper{ // p1	      |	  "v1": *litter_test.CustomMultiLineDumper{ // p0
    	            	    multi							    multi
    	            	    line							    line
    	            	  },								  },
    	            	  "v2": p1,						      |	  "v2": p0,
    	            	  "v2x": *litter_test.CustomMultiLineDumper{			  "v2x": *litter_test.CustomMultiLineDumper{
    	            	    multi							    multi
    	            	    line							    line
    	            	  },								  },
    	            	  "v3": litter_test.CustomSingleLineDumper<custom>,		  "v3": litter_test.CustomSingleLineDumper<custom>,
    	            	  "v4": *litter_test.CustomSingleLineDumper<custom>, // p0    |	  "v4": *litter_test.CustomSingleLineDumper<custom>, // p1
    	            	  "v5": p0,						      |	  "v5": p1,
    	            	  "v6": *litter_test.CustomSingleLineDumper<custom>,		  "v6": *litter_test.CustomSingleLineDumper<custom>,
    	            	}								}
FAIL
exit status 1
FAIL	github.com/sanity-io/litter	0.021s
```

**Comment by erikgrinaker** [comment]
Looks like some sort of race condition; running the test multiple times fails randomly.

**Comment by erikgrinaker** [comment]
This was fixed in c839d6f7e30134ee8a7d696eb053c98abc8fb8be
